README.md

```diff
@@ -103,7 +93,7 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
 * [@theninjaprawn](https://twitter.com/theninjaprawn) for the patchfinder64 additions
 * [@saurik](https://twitter.com/saurik) for Cydia and Substrate
 * [@FCE365](https://twitter.com/FCE365) for the empty_list reliability improvements
-* Credits for [Undecimus-Resources](https://github.com/pwn20wndstuff/Undecimus-Resources)
+#### Credits for [Undecimus-Resources](https://github.com/pwn20wndstuff/Undecimus-Resources)
```

Comment:
Link to the repo contributor page for that repo.

Reply:
done